src/context.ts

```diff
@@ -37,12 +38,16 @@ export async function getInputs(): Promise<Inputs> {
     config: core.getInput('config'),
     configInline: core.getInput('config-inline'),
     append: core.getInput('append'),
-    cleanup: core.getBooleanInput('cleanup')
+    cleanup: core.getBooleanInput('cleanup'),
+    keepState: core.getBooleanInput('keep-state')
   };
 }
 
-export async function getBuilderName(driver: string): Promise<string> {
-  return driver == 'docker' ? await Docker.context() : `builder-${uuid.v4()}`;
+export function getBuilderName(name: string, driver: string): string {
+  if (name) {
+    return name;
+  }
+  return driver == 'docker' ? 'default' : `builder-${uuid.v4()}`;
```

Comment:
Why changing `await Docker.context()` to `default`? It might not always be default so we should keep that.